deploy/cephcsi/image/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM ceph/ceph:v14.2
 LABEL maintainers="Ceph-CSI Authors"
 LABEL description="Ceph-CSI Plugin"
 
+# Removing ceph-iscsi repo to workaround the repo issue while upgrading
+RUN rm -f /etc/yum.repos.d/ceph-iscsi.repo && yum -y update && yum clean all
```

Comment:
deleting the `.repo` file is not appropriate. without it, it will be difficult to inspect the container and see where the installed packages come from.

It is better to `--disablerepo=<name>` where `<name>` is the name listed in the `.repo` file. The option can be used multiple times in case needed.

Reply:
@nixpanic before removing I tried to disable the repo by the options it provided it was not working for me, so the removal. Is it working for you if you disable the repo file ?

Reply:
It should, we do that with other containers and installation scripts too. I have not tried it with this container yet, and I do not know what the contents of `ceph-iscsi.repo` so I can not give an immediate example of how to structure the yum command.

Reply:
>
  yum --disablerepo=ceph-iscsi-noarch ...

     4. Disable the repository permanently, so yum won't use it by default. Yum
        will then just ignore the repository until you permanently enable it
        again or use --enablerepo for temporary usage:

            yum-config-manager --disable ceph-iscsi-noarch
        or
            subscription-manager repos --disable=ceph-iscsi-noarch

>

These are the main options, tried these and its failing.

Reply:
Do you have the exact snippet of the `Dockerfile` and the error messages of how/when it fails?

Reply:
```
sh-4.2# grep -e '^\[' -e ^enabled /etc/yum.repos.d/ceph-iscsi.repo 
[ceph-iscsi]
enabled=1
[ceph-iscsi-noarch]
enabled=1
[ceph-iscsi-source]
enabled=1
```
So the command to disable the repository would be:
```
yum -y --disablerepo='ceph-iscsi*' update
```